src/MessagingThreadHistory/MessagingThreadHistory.less

```diff
@@ -2,7 +2,7 @@
 
 .ThreadHistory--Container {
   .padding--right--s();
-  overflow-y: overlay;
+  overflow-y: auto;
```

Comment:
Better support for this. `overlay` isn't really preferable, either.

Reply:
Good call. I think I read overlay was deprecated in favor of auto, and then possibly brought back but not recommeneded.

Reply:
Indeed. It works better if you change that one setting on MacOS to always show scroll, I believe, but that's a hyper-specific case and not worth optimizing for at this point, IMO.